packages/framework/ini-files/config-specs.ini

```diff
@@ -437,6 +437,8 @@ opt-set-cmake-var SuperLU_INCLUDE_DIRS PATH FORCE : ${SUPERLU_INC|ENV}
 opt-set-cmake-var SuperLU_INCLUDE_DIRS STRING FORCE : ${SUPERLU_INC|ENV}
 opt-set-cmake-var SuperLU_LIBRARY_DIRS PATH FORCE : ${SUPERLU_LIB|ENV}
 opt-set-cmake-var SuperLU_LIBRARY_DIRS STRING FORCE : ${SUPERLU_LIB|ENV}
+opt-set-cmake-var SuperLU_LIBRARY_NAMES STRING FORCE : superlu;m
+opt-set-cmake-var ML_ENABLE_SuperLU BOOL FORCE : OFF
```

Comment:
I just noticed that this breaks the existing GCC 8.3.0 nompi build in my PR.  Probably want to move these two lines into the individual configurations (that's what I did at least).